.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+// Placez vos paramètres dans ce fichier pour remplacer les paramètres par défaut et les paramètres utilisateur.
+{
+    "files.associations": {
+        "*.yml": "ansible"
+    }
+}
```

Comment:
Visual Studio Code, the open source Microsoft editor, not that bad :)
This commit can be rejected if you don't want to track editor specific config file (maybe just add .vscode in .gitignore then).